code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -4599,105 +4599,83 @@
 	glass_name = "cup of xuizi pulque"
 	glass_desc = "A variation of Mictlanian pulque that is safe to consume for Unathi."
 
-//ZZZZOOOODDDDAAAAA
-
+// Zo'ra Sodas
 /decl/reagent/drink/zorasoda
-	name = "Zo'ra Soda Cherry"
-	description = "Zo'ra Soda, cherry edition. All good drinks come in cherry."
-	color = "#102000"
-	adj_sleepy = -2
-	caffeine = 0.4
-	taste_description = "electric cherry"
-	carbonated = TRUE
-
-/decl/reagent/drink/zorasoda/phoron
-	name = "Zo'ra Soda Phoron Passion"
-	description = "Reported to taste nothing like phoron, but everything like grapes."
-	color = "#863333"
-	adj_sleepy = -2
-	caffeine = 0.4
-	taste_description = "electric grape"
-	carbonated = TRUE
-
-/decl/reagent/drink/zorasoda/kois
-	name = "Zo'ra Soda K'ois Twist"
-	description = "Whoever approved this in marketing needs to be drawn and quartered."
-	color = "#dcd9cd"
-	adj_sleepy = -2
+	name = "Zo'ra Soda"
+	description = "Zo'ra Soda. You aren't supposed to see this."
+	color = "#000000"
+	adj_dizzy = -6
+	adj_drowsy = -4
+	adj_sleepy = -3
```

Comment:
This seems a bit too strong for a drink that you can buy or get from the loadout.

Reply:
It's on the same level as coffee, but one more. I.e. coffee has dizzy at -5, zo'ra soda has it at -6.